### DISCUSSION TITLE
Lowrey NL30 (Kawai KL-4) with GrandOrge

### DISCUSSION THREAD

**Discussion by TangoFanta** [GrandOrgue/grandorgue#1862] [Help]
Hi Everyone,

A newbie here...sorry.

Is anyone using GrandOrgue with a Lowrey Parade NL30 or Kawai KL-4 organ? I'm new to MIDI other than for a few cheap keyboards. I've mapped the keyboards/pedalboards, but each key from the organ sends 2 sets of velocities for each key press. (0,1 and 64,65). Maybe this is common for a lot of organs, but this is the first one I've had experience with.

Because there is a gap between the two velocity sets, when I map a virtual keyboard to a physical one, the note starts when the key is gently touched, but then stops when I press further down. It starts again when velocity reaches 64 towards the bottom of the key.

Currently I have GrandOrgue set to ignore the key until velocity reaches 64. That stops the key from stopping and starting, but that means the note doesn't play until the key is almost fully pressed down. 

How do I stop the organ from sending the velocity value? If that can't be done, how do I combine both velocities so the note keeps playing when I press further down.

Does anyone have tips for setting up Lowrey/Kawai organs with GrandOrgue? 

Many Thanks
TangoFanta

**Comment by midimusic** [comment]
I suspect your keyboards are sending "second touch" midi events. I have a keyboard which also can send second touch and it can be turned off if I do not want it.

I suggest you look into whether you can also turn it off on your organ (assuming you do not use it). Otherwise I suspect you will have to live with it.

csw900

**Reply by TangoFanta** [reply]
Well at least I know there is such a thing now. For a long time I thought it was a problem with the organ, but even the manual states it will send channels 5,6,7 on each note, and channels 15 and 16 on the pedalboard. There is a "TOUCH" mode, but that sends channel 3 with even more information when it's turned on. So looks like I'm stuck with it for now, but it looks like it's working good enough.

I've set the lowest and highest velocity for each key to 65, so it ignores the first touch of the key. If I could combine both so that the key plays for the full key stroke would be ideal, but the organ sends a velocity of 0, 1, then 0 again until the key is pressed lower. 

Just wondering, with the organ sending so much extra information, would that cause the buffer to fill up faster even though no sounds are played?, or does it just go by the samples that are actually being played? 

Many Thanks
TangoFanta

**Comment by TangoFanta** [comment]
Thanks so much for the feedback. The "second touch" for each key looks to be normal and can't be turned off. The "Touch" mode sends even more events, which we don't need. It would probably be easier to just get a couple of cheaper midi keyboards, but since we have the Lowrey, just as well use it until it stops working. 

I think we have GO working decent enough for our test system. It's an older Xeon with 16GB ram with a Presonus Firebox. Latency is 13-15ms with CPU % around 8-15% max. I've managed to get it as low as 7ms, but start to get stuck keys while playing, unless that's due to something else other than latency.

**Comment by midimusic** [comment]
Glad you now have GO working well enough to use. Don't worry about latency - I have proved that anything up to around 100ms of real measured latency is not detectable when playing.

csw900

**Reply by TangoFanta** [reply]
Thanks. I don't have enough experience with a proper pipe organ to understand all the terminology and options, but our organist is very pleased with what it can do. We were using an electronic piano with pipe organ sound until we found our Lowrey had MIDI. 

We have a Casavant pipe organ from 1935, but repairs are too costly for us to justify what with other fundraisers for our church.  I'm sure we will get it repaired eventually, however GO will be a great alternative until then. 

We are using the Friesach sample set for now. I'm sure we will look at other free and paid sets once we get more comfortable with the software. Right now we just wanted something better than the built in Lowrey sounds, which even the demo sample set is much better. Also, we have some amplifier issues with the organ itself, so MIDI was the only way we could reliabily use the organ. 

Such a wonderful program and so many options. I was anticipating having to look at more expensive options, however we haven't had any major issues with GO yet to have reason to look elsewhere. I'm sure I can tweak settings a bit more, but we are only outputting mono to a keyboard amp and our pa system until we can afford or get donations for a better sound system.

Thanks again
TangoFanta
